app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinUi.kt

```diff
@@ -1,6 +1,6 @@
 package org.simple.clinic.forgotpin.confirmpin
 
-interface ForgotPinConfirmPinUi : UiActions {
```

Comment:
Can we rename the `UiActions` interface to be prefixed with the `ForgotPinConfirmPin` name?

Reply:
Will do 👍🏾